ingest-app/project.clj

```diff
@@ -4,8 +4,10 @@
   :dependencies [[camel-snake-kebab "0.4.2"]
                  [clj-http "2.3.0"]
                  [com.draines/postal "2.0.3"]
+                 [jakarta.servlet/jakarta.servlet-api "4.0.4"] ;;5.x and 6.x did not work
```

Comment:
what is using this library?

Reply:
we can further investigate this in https://bugs.earthdata.nasa.gov/browse/CMR-10550